lib/cc/analyzer/issue_relative_path_validation.rb

```diff
@@ -11,6 +11,13 @@ def message
 
       private
 
+      def relative_to?(directory)
```

Comment:
> I'm interested in having an in-chat conversation about how we have decided / are deciding to "address symlinks". I think there's some nuance there and I know we've fought this battle before.

Definitely worth having a discussion (maybe in-person tomorrow at this point). I think our hand is being forced a bit now: we've fought parts of this battle, but also deferred the war (so to speak). With these stricter validations, and the behavior of content-based-fingerprinting, we now *must* "handle" symlinks in one way or another.